### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 3

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#453] [Esami & Esoneri M-Z]
## Traccia

<img width="627" alt="image" src="https://github.com/user-attachments/assets/ff3ca6c9-c818-4aee-8da8-e975b5c9b46e">

**Comment by rimaout** [comment]
<img width="627" alt="image" src="https://github.com/user-attachments/assets/53a5fd72-1171-4187-898f-797e2e75ecb5">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/6aeadeb6-cd6b-4dd0-ac55-8e9be94cb431)
